website/docusaurus2-graphql-doc-generator/README.md

```diff
@@ -0,0 +1,128 @@
+# GraphQL Documentation Generator for Docusaurus 2
+
+[![npm](https://img.shields.io/npm/dt/@edno/docusaurus2-graphql-doc-generator?style=flat-square)](https://www.npmjs.com/package/@edno/docusaurus2-graphql-doc-generator)
+[![Latest Version](https://img.shields.io/npm/v/@edno/docusaurus2-graphql-doc-generator?style=flat-square)](https://www.npmjs.com/package/@edno/docusaurus2-graphql-doc-generator)
+[![GitHub license](https://img.shields.io/github/license/edno/docusaurus2-graphql-doc-generator?style=flat-square)](https://raw.githubusercontent.com/edno/docusaurus2-graphql-doc-generator/main/LICENSE)
+
+This plugin generates a **Markdown documentation** from a **GraphQL schema**.
+
+The documentation is generated for **Docusaurus 2** [docs feature](https://v2.docusaurus.io/docs/docs-introduction).
+
+## Installation
+
+### npm
+
+```shell
+npm install --save @edno/docusaurus2-graphql-doc-generator
+```
+
+### Yarn
+
+```shell
+yarn add @edno/docusaurus2-graphql-doc-generator
+```
+
+Then you add it in your site's `docusaurus.config.js`'s plugins option:
```

Comment:
Then you need to add it to the plugin options in `docusaurus.config.js` of your site.